src/specialMenus.js

```diff
@@ -904,8 +917,7 @@ WindowThumbnail.prototype = {
     this.actor.connect('leave-event', ()=>{
       if (!this.isFavapp) {
         this._hoverPeek(OPACITY_OPAQUE, this.metaWindow, false)
-        this.actor.remove_style_pseudo_class('outlined')
```

Comment:
Can you re-add these lines? This is causing the thumbnails to enlarge when cycling Super+#.